core/server/data/import/data-importer.js

```diff
@@ -133,6 +133,12 @@ DataImporter.prototype.doImport = function (data) {
                         if (results) {
                             importResults = importResults.concat(results);
                         }
+
+                        return utils.importSubscribers(tableData.subscribers, t);
+                    }).then(function (results) {
+                        if (results) {
+                            importResults = importResults.concat(results);
+                        }
```

Comment:
yes 👍 `onFulFilled` callback is resolved with what is returned by it. Bluebird and ES6 Promises behave the same.